io.missioncenter.MissionCenter.json

```diff
@@ -0,0 +1,169 @@
+{
+  "app-id": "io.missioncenter.MissionCenter",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "44",
+  "sdk": "org.gnome.Sdk",
+  "command": "missioncenter",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=all",
+    "--socket=wayland",
+    "--talk-name=org.freedesktop.Flatpak",
+    "--talk-name=org.gnome.Settings",
+    "--system-talk-name=org.freedesktop.systemd1",
+    "--system-talk-name=org.freedesktop.NetworkManager",
+    "--filesystem=xdg-data/flatpak/exports/share:ro",
+    "--filesystem=/var/lib/flatpak/exports/share:ro",
+    "--filesystem=xdg-data/flatpak/app:ro",
+    "--filesystem=/var/lib/flatpak/app:ro",
+    "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/share:/var/lib/flatpak/exports/share:~/.local/share/flatpak/exports/share"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "missioncenter",
+      "buildsystem": "simple",
+      "build-options": {
+        "append-path": "/run/build/missioncenter/bin",
+        "env": {
+          "RUST_BACKTRACE": "full",
+          "RUSTUP_HOME": "/run/build/missioncenter",
+          "CARGO_HOME": "/run/build/missioncenter/cargo"
+        }
+      },
+      "build-commands": [
+        "rustup default stable-x86_64-unknown-linux-gnu",
```

Comment:
why no using the rust sdk. you shouldn't need rust.

Reply:
I need to use a version of Rust that has both x86_64-unknown-linux-gnu and x86_64-unknown-linux-musl and is the exact same version, the default one provided by the sdk extension does not have musl

Reply:
why would you need musl?

Reply:
The `proxy` module.
This builds a statically linked executable that is run on the host to read information from /proc

Reply:
if by "on the host" you mean out of the sanbox? How will that work?

Reply:
The main executable launches the `missioncenter-proxy` binary using `flatpak-spawn`. This binary goes through `/proc`, reads all the information it requires for running processes, serializes this data and prints it to `stdout`. The main process reads this data back, and presents it to the user (like in the above screenshot).

Reply:
and how does that works? `flatpak-spawn --host` require the command to be on the host.